### ISSUE TITLE
if gnatsd supports IPV6?

### ISSUE THREAD

**Issue by huangyigui** [nats-io/gnatsd#667]
- [ ] Feature Request or Change Proposal
 
#### Use Case:
gnatsd should support IPV6
 
#### Who Benefits From The Change(s)?
IPV6 stack is a standard config for mostly main software,so nats also should support it

**Comment by kozlovic** [comment]
NATS Server does support IPv6. For instance running gnatsd like this:
```
gnatsd -a "::" -p 4223
```
Then connecting a client to this port and do a netstat on the server:
```
$ netstat -an | grep 4223
tcp6       0      0  ::1.4223               ::1.63114              ESTABLISHED
tcp6       0      0  ::1.63114              ::1.4223               ESTABLISHED
tcp46      0      0  *.4223                 *.*                    LISTEN     
```
Note however, that in configuration file, if you were to specify the `listen` option, you would need to use `[` and `]` for the IPv6 address. For instance:
```
listen: "[::]:4223"
```
If you run in any issue, please don't hesitate to file an issue on GH.

**Comment by huangyigui** [comment]
thanks for your reply

**Comment by tuorifei** [comment]
execuse me
we have the cluster configure below:
```
cluster {
   listen:"[::]:1071"
  # Routes are actively solicited and connected to from this server.
  # Other servers can connect to us if they supply the correct `credentials`
  # in their routes definitions from above.
  routes = [ 
"nats-route://[9999::101]:1071"
"nats-route://[9999::102]:1071"
"nats-route://[9999::103]:1071"
  ]
}
```
but there be some probles:
`Parse error on line 39: 'Expected a map value terminator "," or a map terminator "}", but got ':' instead.'`
could you tell me  what the route url right usage？
thanks !!!

**Comment by kozlovic** [comment]
Which version are you using? I have no problem using this config with `v1.2.0`.

**Comment by tuorifei** [comment]
Thanks。
We used v1.0.2 of gnatsd。we fix the problem using v1.0.4 today。thanks again!
